### PR TITLE
chore(stacks): Add Docker QA Stack for 7.21.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by psavidis**
Related-to: camunda/camunda-bpm-platform/issues/3339

### CODE REVIEW

**Comment by tmetzke** [comment]
Thanks, @psavidis! Two things to consider:

* Did you stick to https://github.com/camunda/portainer-templates#how-to-create-a-new-stack-type-template? I cannot see the adjustments in all the files mentioned there.
* With 7.20, we drop support for WAS9 (see [HERE](https://docs.camunda.org/enterprise/announcement/#camunda-platform-7-20)). I think we can skip creating a new stack for this.

**Comment by psavidis** [comment]
> Thanks, @psavidis! Two things to consider:
> 
> * Did you stick to https://github.com/camunda/portainer-templates#how-to-create-a-new-stack-type-template? I cannot see the adjustments in all the files mentioned there.
> * With 7.20, we drop support for WAS9 (see [HERE](https://docs.camunda.org/enterprise/announcement/#camunda-platform-7-20)). I think we can skip creating a new stack for this.

- Hey, thanks for reminding me of the guide. I've reverted everything and started from scratch with a clean setup (i was missing the python command).
- Also, didn't include WAS 9 as recommended.

File: new-stack-templates.json [hunk 1]
```diff
@@ -408,6 +408,41 @@
...
+        "name": "DATABASE_NAME"
+      },
+      {
+        "default": "oracle",
+        "label": "Database Type (oracle, postgres, h2)",
+        "name": "DATABASE_TYPE"
+      }
+    ],
+    "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/registry.png",
+    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-729-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml</a>",
```
Comment (tmetzke) [inline new-stack-templates.json]: ❌ Looks like a `7.19` copy-paste error that also needs to be corrected for the `7.20` entry above.
```suggestion
    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml</a>",
```

File: stack-templates.json [hunk 2]
```diff
@@ -1899,5 +1934,69 @@
...
+    "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/registry.png",
+    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/camunda-camunda-bpm-platform-tomcat-7.5.0/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/camunda-camunda-bpm-platform-tomcat-7.5.0/docker-stack.yml</a>",
+    "platform": "linux",
+    "repository": {
+      "stackfile": "stacks/camunda-camunda-bpm-platform-tomcat-7.5.0/docker-stack.yml",
+      "url": "https://github.com/camunda/portainer-templates"
+    },
+    "title": "Camunda Platform 7.5.0 Tomcat",
+    "type": 2
   }
```
Comment (tmetzke) [inline stack-templates.json]: 🔧 I'd probably rather remove those entries from the `new-stack-templates.json` so they are not generated anymore.

**Review by tmetzke — CHANGES_REQUESTED** [review]
👍 The addition itself looks good.
🔧 There are some minor things to change here that make this more consistent.

File: stack-templates.json (hunk 2 above)
Comment (psavidis) [reply stack-templates.json]: Done.

File: stack-templates.json [hunk 3]
```diff
@@ -399,7 +399,7 @@
       }
     ],
     "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/registry.png",
-    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-720-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-720-latest/docker-stack.yml</a>",
+    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-729-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-720-latest/docker-stack.yml</a>",
```
Comment (tmetzke) [inline stack-templates.json]:
```suggestion
    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-720-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-720-latest/docker-stack.yml</a>",
```

File: stack-templates.json [hunk 4]
```diff
@@ -408,6 +408,41 @@
...
+        "name": "DATABASE_NAME"
+      },
+      {
+        "default": "oracle",
+        "label": "Database Type (oracle, postgres, h2)",
+        "name": "DATABASE_TYPE"
+      }
+    ],
+    "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/registry.png",
+    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-729-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml</a>",
```
Comment (tmetzke) [inline stack-templates.json]:
```suggestion
    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml</a>",
```

**Review by tmetzke — APPROVED** [review]
🔧 I think you can remove the new tomcat stacks, we won't need them.
👍 Other than that, we're good to go I think

**Pull request merged**